snmp/tests/compose/data/aruba-switch.snmprec

```diff
@@ -10,6 +10,12 @@
 1.3.6.1.4.1.14823.2.2.1.1.1.4.0|2|1
 1.3.6.1.4.1.14823.2.2.1.1.1.5.0|64x|ac174403
 1.3.6.1.4.1.14823.2.2.1.1.1.7.0|2|6
```

Comment:
> We cannot add the metrics to test_profiles because this test runs the python check. We don’t currently have per-profile tests using the corecheck

We actually have some per profile core check tests here:

https://github.com/DataDog/integrations-core/blob/master/snmp/tests/test_e2e_core.py

Reply:
We were considering that we should change the e2e testing strategy for core check, we are diverging more and more, having e2e tests in integrations core has some cons. Should we leverage s8s instead ?

Reply:
Anyhow, added a ticket to work on this, assigned to myself

Reply:
> We were considering that we should change the e2e testing strategy for core check, we are diverging more and more, having e2e tests in integrations core has some cons. Should we leverage s8s instead ?

There is definitely a discussion to start since we don't want to depend on integrations-core in mid/long term for SNMP corecheck.

> Should we leverage s8s instead ?

Not sure about that, ideally we should avoid tests depend on external system when possible.

> This is more an integration test than an e2e test, isn't integrations tests on datadog agent enough ?

Testing in datadog-agent repo can be enough, but I think we still need to implement testing with `snmpsim` for profile based testing.
A requirement for moving profile based testing in datadog-agent is to move SNMP profiles to datadog-agent.

Reply:
What about using s8s for profile based tests ? This would be closer to the end user experience, and would be a full end to end test.